### PR TITLE
Batchnorm track running state

### PR DESCRIPTION
**Pull request by ferhah**
I found out that during inference a larger batch size results in better results than a smaller one. To avoid this the batchnorm layer needs to track the running state.

### CODE REVIEW

**Comment by iamsimha** [comment]
Hi

We are using pytorch batchnorm instead of torchmeta batch norm

https://github.com/tristandeleu/pytorch-maml/blob/c7d994a3e9900d3d6790dbe921cd63abbc6589d4/maml/model.py#L11

Does the inner loop gradient descent update batch norm parameters?

PS: I am not the author. I am trying to understand the implementation.
